### DISCUSSION TITLE
Equipo para hackathon

### DISCUSSION THREAD

**Discussion by Israel-12-u** [Launch-X-Latam/MisionFrontEnd#502] [Team Posts]
Hola, busco equipo para hackathon, tengo conocimientos en lenguajes como HTML,JS,Python,Java y C++, además tengo conocimientos en bases de datos como MySQL,POSTGREST,MongoBDn,SQLite3 y se usar framewroks para la creación de APIS
Dejo mi correo para cualquier contacto: lx04770@innovaccion.mx
Cuentas de GitHub: Israel-12-u  ó Kevin12-0

**Comment by igutisan** [comment]
Hola, quieres unirte a mi grupo?

**Comment by IsaiCeSa** [comment]
.

**Comment by DanielBustillosVillar** [comment]
Buenas, a mi me gustaria unirme a tu grupo mi correo es: lx05538@innovaccion.mx

**Comment by EliasQuino** [comment]
A mi me agregan: lx08662@innovaccion.mx

**Comment by giankpetrov** [comment]
lx09705@innovaccion.mx @Israel-12-u  puedes ver mi repositorio si das click en mi perfil y contact
